procurement/filters.py

```diff
@@ -90,7 +89,8 @@ def filter_region(self, queryset, name, value):
         countries = [x[0] for x in Council.COUNTRY_CHOICES]
         # TODO: Wales + Scotland
         if value in countries:
-            return queryset.filter(**{"council__nation": value})
+            print(queryset.filter(council__nation=value))
```

Comment:
debug?

Reply:
Oops. That;s been removed, and the return has been formatted to look like the others.